core/src/commonMain/kotlin/entity/Emoji.kt

```diff
@@ -48,21 +48,17 @@ public class StandardEmoji(override val name: String) : Emoji {
 }
 
 /**
- * An instance of an [emoji](https://discord.com/developers/docs/resources/emoji#emoji-object) belonging to a specific
- * [Guild].
+ * Superclass for all non standard emojis.
```

Comment:
The comment on `Emoji` should be updated to say what the subtypes are.

Reply:
It's already mentioned using @see, so why mention it twice

Reply:
I was referring to this line: https://github.com/kordlib/kord/blob/d23b8c5a2658ddde7e4e367d5ac8397f62ac1928/core/src/commonMain/kotlin/entity/Emoji.kt#L21